steps/check_changed_files.yml

```diff
@@ -0,0 +1,62 @@
+# Usage:
+# This template checks changed files against the main development branch and sets an output variable for a certain path filter to indicate changes
+# at this location. This variable can then be used in subsequent jobs, for example, to set the condition on which the job should run.
+#
+# Example:
+# jobs:
+#   - job: A
+#     steps:
+#       - template: steps/check_changed_files.yml@templates
+#         parameters:
+#           pathFilters:
+#             - name: source_code
+#               path: src
+#  - job: B
+#    condition: and(succeeded(), eq(dependencies.A.outputs['checkfiles.source_code'], 'true'))
```

Comment:
This should be "check_files" and it's probably worth adding the parameter to the step above to make it clearer what this is.

Reply:
Done